service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -376,6 +379,14 @@ public static List<String> validateRequestInputsAndOutputs(
     return errorList;
   }
 
+  public static List<String> validateRequestContainsCallCachingParameter(RunSetRequest request) {
+    List<String> errorList = new ArrayList<>();
+    if (request.isCallCachingEnabled() == null) {
+      errorList.add("Request must contain boolean call_caching_enabled parameter");
+    }
+    return errorList;
+  }
+
```

Comment:
Mild preference for backwards compatible API that defaults to... on or off, idk

Reply:
Defaulted to `off` as we do for GCP.